contexts/audit/ruby/Makefile

```diff
@@ -0,0 +1,14 @@
+-include ../../prerequisites/tasks.mk
+-include ../../prerequisites/ruby.mk
+
+##>> run bundle audit
+
+.DEFAULT_GOAL := all
+.PHONY: all
+
+## all (default): run bundle audit
+all: bundle
+	@cd $(base_path) && bundle exec bundle audit --update
```

Comment:
Why is there `bundle exec bundle` here and in line 14 ?

Reply:
I changed the way to ignore advisories by creating a file name `.bundler-audit.ignore`, so now we have just one target. Please evaluate again all this file.